### ISSUE TITLE
Support for localization / translation of notification ?

### ISSUE THREAD

**Issue by MaZZly** [google/physical-web#938]
Hello,

Is there any support for making localized notifications?
e.g. add some magical meta tags with lang="en" for english lang? and the device would pick the most valid one. 

For our beacons we can easily make the notification localized with a url-param, but would be awesome if we could use some magical meta tags to have one beacon that provides all languages.
(In Finland we have 2 official languages, 3 if you count English which should in our case also be included.)

Cheers!

**Comment by ferencbrachmann** [comment]
As far as we understand that's not possible. 
However, we do support multi-language content with [Beeem](https://beeem.co) and when this becomes possible we'll support it.

**Comment by MaZZly** [comment]
**Hmm that's a bit dissapointing honestly..**

Multi-language content on our own pages is a cakewalk, just the problem of getting a user to click on a notification that is not in their native language.. (Think: why would I click a notification in some weird language that I don't neccesarily understand?)

I would think this should be relatively easy to support with either something like hreflang (so the metadata could be crawled by the device for preferred language):
```html
<link rel="alternate" href="http://example.com/en" hreflang="en" />
<link rel="alternate" href="http://example.com/sv" hreflang="sv" />
```
Or being able to place multiple titles/descriptions in a single page's <head> like:
```html
<meta name="description" content="This is an example text" lang="en">
<meta name="description" content="Detta är en exempeltext" lang="sv">
```


I find it hard to believe that this would have been missed and is not supported, considering how important localized content is to a user.
Not to mention how common it is for countries to be multilingual https://en.wikipedia.org/wiki/List_of_multilingual_countries_and_regions

**Comment by auge2u** [comment]
@MaZZly 
I agree.  This is exactly the standards we are going to add to the next version of Physical Web we are currently laying the foundation to build. We will need people like you to be part of the process.

Join the slack channel with the link below where we are upgrading the current and Physical Web standards. Our group is spearheading the initiatives for adoption, spam solutions, and commercial and public partnerships to implement PW2 as a ubiquitous solution for proximity engagement.
  
https://join.slack.com/t/physicalweb2/shared_invite/enQtMzMyMDYyMTE2NjE1LTA0NWRiNzdjYTM3NTFiODhjN2MxODMwMWM1NzhmZTE3YzRiZjA2ZTQyMTcxMGZkYzg3ODc3MWUwZDY5YWJjMDA

**Comment by adriancretu** [comment]
If you know where you place your beacon what's the point of a multi-language meta schematics? Just use an URL that resolves to the localized language of where the beacon is. If you use a www internet page meant for all the people on the planet with 300 meta tags and i18n support for all your users, then you've got the PW philosophy wrong. Use a unique URL to solve for the beacon's geo, not the other way around.

**Comment by ferencbrachmann** [comment]
@adriancretu There is a huge opportunity for multi-language in museums and in tourism all over Europe.

**Comment by MaZZly** [comment]
@adriancretu because e.g. in Finland we have 2 main languages, and we can't with any certainty know if the user would be a native Finnish or Swedish speaker (or even English expat).
It's not about cramming 300 meta tags for all languages, but at least making it usable for the region where it is placed.

And exactly like @ferencbrachmann said, there are many cases where some major languages would be useful, e.g. showing metadata in correct language at events/museum. simply placing English, Spanish, Mandarin and some other languages and you would cover a large portion of the potential population.

**Comment by MaZZly** [comment]
@auge2u I tried the link but never got any mail from Slack.. tried again now but still same thing, anything that needs to be approved on your end first?

**Comment by auge2u** [comment]
@MaZZly 
Odd that it didn't work. Try again below? 

[https://join.slack.com/t/physicalweb2/shared_invite/enQtMzMyMDYyMTE2NjE1LTA0NWRiNzdjYTM3NTFiODhjN2MxODMwMWM1NzhmZTE3YzRiZjA2ZTQyMTcxMGZkYzg3ODc3MWUwZDY5YWJjMDA](url)

**Comment by ravipratapm** [comment]
@MaZZly Actually, multi-language notifications ARE supported by Google Nearby, and you can take full advantage of them with [Beaconstac](https://www.beaconstac.com).

**Comment by ravipratapm** [comment]
@MaZZly If you want more information on how it's done, please see this: https://developers.google.com/nearby/notifications/developer_faq (see the last question "What about localization of the title Nearby Notifications uses?").

**Comment by MaZZly** [comment]
@ravipratapm does that work also with eddystone URLs? I don't think it's possible to provide an attachment there?

**Comment by ravipratapm** [comment]
@MaZZly That's right - it's not available on the Physical Web. Attachments are a feature of Google Nearby.

**Comment by MaZZly** [comment]
@ravipratapm okay, so then this Issue is still very valid.

**Comment by MaZZly** [comment]
@auge2u btw still doesn't work.. just says email has been sent to my gmail, but can't find any mails in inbox/spamfolder etc

**Comment by auge2u** [comment]
@MaZZly Other people are signing up but how about you send me an email at contactus@phwa.io ?
should catch you there.

**Comment by MaZZly** [comment]
@auge2u sent an email there now =)
